cms/djangoapps/contentstore/views/course.py

```diff
@@ -896,6 +915,42 @@ def _config_course_advanced_components(request, course_module):
     return filter_tabs
 
 
+# pylint: disable=invalid-name
+def _config_course_settings(request, course_module, filter_tabs=True):
+    """
+    Check to see if the user enabled some advanced settings (boolean).
```

Comment:
this has been reworked in https://github.com/edx/edx-platform/commit/c384ee50b77e52fdf5cf78b7e3caa5f04fe5eef2